.github/workflows/continuous-integration.yaml

```diff
@@ -58,7 +58,6 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 12
           - 14
-          - 15
           - 16
+          - 18
```

Comment:
Since we are doing `"node": ">=16 <19"` we could also ditch the v14 here right?

Reply:
Ah yes I missed this one, thanks!